From ec70096d6d8dd8f12839facbac5333e961d28b04 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Wed, 1 Aug 2012 22:25:01 +0200
Subject: [PATCH 1/2] Expose module directory in pkg-config configuration file

---
 src/config/nyx.pc.in |    1 +
 1 file changed, 1 insertion(+)

diff --git a/src/config/nyx.pc.in b/src/config/nyx.pc.in
index 80392b3..2903c02 100644
--- a/src/config/nyx.pc.in
+++ b/src/config/nyx.pc.in
@@ -2,6 +2,7 @@ prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 includedir=${prefix}/include
+moduledir=@NYX_MODULE_DIR@
 
 Name: Nyx
 Description: Open webOS Portability Layer
-- 
1.7.9.5

